rust/protocol/src/fingerprint.rs

```diff
@@ -41,8 +41,11 @@ fn get_encoded_string(fprint: &[u8]) -> Result<String> {
     }
 
     let s = fprint.chunks_exact(5).take(6).map(read5_mod_100k).fold(
-        String::with_capacity(5*6),
-        |mut s, n| {write!(s, "{:05}", n).ok(); s},
+        String::with_capacity(5 * 6),
+        |mut s, n| {
+            write!(s, "{:05}", n).expect("can always write to a Vec");
```

Comment:
Nitpick: String, not Vec!

Reply:
Oh, sorry! I copied the message from a different expect